pkg/logs/launchers/journald/launcher.go

```diff
@@ -78,12 +85,27 @@ func (l *Launcher) run() {
 				log.Warn(identifier, " is already tailed. Use config_id to tail the same journal more than once")
 				continue
 			}
+
+			if source.Config.Path != "" {
+				// Add path to flare if specified in configuration
+				allJournalSources = append(allJournalSources, source.Config.Path)
+			} else {
+				// Check default locations otherwise
+				if _, err := os.Stat("/var/log/journal"); err == nil {
+					allJournalSources = append(allJournalSources, "/var/log/journal")
+				} else if _, err := os.Stat("/run/log/journal"); err == nil {
+					allJournalSources = append(allJournalSources, "/run/log/journal")
+				}
+			}
+
 			tailer, err := l.setupTailer(source)
 			if err != nil {
 				log.Warn("Could not set up journald tailer: ", err)
 			} else {
 				l.tailers[identifier] = tailer
 			}
+
+			l.fc.SetAllJournalFiles(allJournalSources)
```

Comment:
It is possible for there to be multiple journald instances. This is an edge case as most users would not configure it this way, but it's possible to have multiple journald tailers with different paths. 
[See our docs here](https://docs.datadoghq.com/integrations/journald/?tab=host#tailing-the-same-journal-multiple-times)

The way the code is written now, the last scheduled config will win, and override the other. We should try to account for this. Due to the nature of how journald paths are configured, you can probably just append to the list instead of override the whole list each time. But i'd double check that this doesn't have other implications.